tests/software/util.py

```diff
@@ -58,8 +58,8 @@ def _chipsec_util(self, arg, helper_class=mock_helper.TestHelper):
         It verifies that no error is being reported. self.log will be populated
         with the output.
         """
-        oshelper._helper =  helper_class()
```

Comment:
looks like oshelper._helper is no longer used.  should we remove it from setUp() and tearDown() as well?

Reply:
removed